swal-forms.js

```diff
@@ -219,7 +219,8 @@
         return inputTag + labelTag
 
         function toHtmlOptions (optionsString, option) {
-          return optionsString + t("<option value='{value}'>{text}</option>", option)
+            option.selected = ((typeof option.selected !== 'undefined') && option.selected) ? ' selected' : ''
```

Comment:
I asked this because the template already contains a space before {selected}: "<option value='{value}' {selected}>{text}</option>". I think it would be one way or the other, right?